warehouse/query-core/src/main/java/datawave/query/jexl/visitors/RewriteNullFunctionsVisitor.java

```diff
@@ -190,7 +160,11 @@ private JexlNode rewriteFilterFunction(FunctionJexlNodeVisitor visitor) {
             default:
                 // multi fielded case
                 rebuiltMultiFieldedFunction = true;
-                return JexlNodeFactory.createOrNode(children);
+                if (visitor.name().equals(IS_NULL)) {
```

Comment:
this seems odd to be doing OR for everything other than IS_NULL and AND for IS_NULL. Hmmm....